lib/edoxen/approval.rb

```diff
@@ -15,3 +15,43 @@
 #   majority
 #   minority
 # }
+require "shale"
+require "shale/type/value"
+module Edoxen
+  class ConsiderationTypeEnumInvalidValue < TypeError; end
+
+  class ApprovalDegreeEnumInvalidValue < TypeError; end
+
+  class ConsiderationTypeEnum < Shale::Type::Value
+    ALLOWED_VALUES = ['affirmative', 'negative'].freeze
+
+    def self.cast(value)
+      if ALLOWED_VALUES.include?(value.to_s)
```

Comment:
As above.

Reply:
Hey, so this was corrected almost as soon as you've said this. But as I've said, I kind of can't contact Marcin

@webdev778 what about you man? I'm sorry we didn't get along the first time around, but I'm getting really worried about him